docs/providers/google.md

```diff
@@ -70,7 +70,7 @@ const options = {
   callbacks: {
     async signIn(user, account, profile) {
       if (account.provider === 'google' &&
-          profile.verified_email === true &&
+          profile.email_verified === true &&
           profile.email.endsWith('@example.com')) {
         return true
```

Comment:
This should be simplified to:
```js
signIn({ profile }) {
  return profile.email_verified && profile.email.endsWith('@example.com')
}
```

The `account.provider === "google"` check is unnecessary because if the user has multiple providers in their config, only the google one would succeed. This is also a value coming from `next-auth`, and it doesn't contribute to anything really.

Do you agree?